lib/functions/RegularFunctions.ts

```diff
@@ -171,50 +170,50 @@ const greaterThanEqual = {
 // https://www.w3.org/TR/sparql11-query/#func-rdfTerms
 // ----------------------------------------------------------------------------
 
-const isIRI = {
+const rdfTermIsIRI = {
```

Comment:
Same as above, rename is not needed?

Reply:
This one is indeed not needed, but, since there was also a `toString` function (now `rdfTermToString`) I renamed all the functions so they still have the same naming style.

Reply:
In this case I would not rename any, except `toString` and just document that it's an en exception. I would say use `termToString` (rdf is unneeded here), `toString_`, or even `str` which is what it's called in SPARQL.